### PR TITLE
Fix codec integration

### PR DESCRIPTION
**Pull request by developit**
With these fixes in place I have the build running locally on a chromebook.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,6 +85,7 @@
     "classnames": "^2.2.5",
     "material-components-web": "^0.32.0",
     "material-radial-progress": "git+https://gist.github.com/02134901c77c5309924bfcf8b4435ebe.git",
+    "mozjpeg": "^6.0.0",
```
Comment (surma) [inline package.json]: Why is this needed? And if it is, would it make more sense to have it in the `codecs/mozjpeg_enc/package.json`?

File: src/lib/codec-wrappers/codec.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 export interface Encoder {
-  encode(data: ImageData): Promise<ArrayBuffer>;
+  encode(data: ImageData): Promise<ArrayBuffer | SharedArrayBuffer>;
```
Comment (surma) [inline src/lib/codec-wrappers/codec.ts]: Iiiinteresting. I removed the `SharedArrayBuffer` and it still compiled. 

I just cleaned the entire repo from build artifacts and now I am seeing the same error. I guess we have some caching in the build system somewhere.

Comment (surma) [reply src/lib/codec-wrappers/codec.ts]: On second thought: I’d actually prefer to enforce `ArrayBuffer` as wasm cannot run on `SharedArrayBuffer`, so it’s not really a possibility anyways.

@jakearchibald: I now remember why this was a thing before: The wasm/Emscripten modules export `HEAP8`, `HEAP16` etc as views onto the memory, which are `Int8Array`, `Int16Array` etc. So `HEAP8.buffer` has type `ArrayBuffer | SharedArrayBuffer`. 

I’ll push a commit to this branch.

**Comment by surma** [comment]
I’m gonna go ahead and merge this. Please still take a quick look at that question of mine, @developit  :D

File: package.json (hunk 1 above)
Comment (surma) [reply package.json]: Gotcha. I’ll remove it.

**Comment by developit** [comment]
Fixed on master.

File: package.json (hunk 1 above)
Comment (surma) [reply package.json]: ... or you will :D
